seqr/views/utils/json_to_orm_utils.py

```diff
@@ -1,35 +1,50 @@
 import logging
+from django.utils import timezone
 
-from seqr.model_utils import find_matching_xbrowse_model, convert_seqr_kwargs_to_xbrowse_kwargs, update_seqr_model
+from seqr.model_utils import update_seqr_model
 from seqr.views.utils.json_utils import _to_snake_case
 
 logger = logging.getLogger(__name__)
 
 
 def update_project_from_json(project, json, verbose=False):
 
-    _update_model_from_json(project, json, verbose=verbose)
+    update_model_from_json(project, json, verbose=verbose)
 
 
 def update_family_from_json(family, json, verbose=False, user=None, allow_unknown_keys=False):
-
-    _update_model_from_json(family, json, user=user, verbose=verbose, allow_unknown_keys=allow_unknown_keys)
+    update_model_from_json(
+        family, json, user=user, verbose=verbose, allow_unknown_keys=allow_unknown_keys, restricted_keys=['pedigree_image']
```

Comment:
no because they also are used to restrict read access on some users

Reply:
I only see `restricted_keys`  in `update_model_from_json` where it's used to raise `Cannot edit field `. How does it restrict read access?

Reply:
Sorry, I was confusing `restricted_keys` with `internal_json_fields` which are checked both on write and read for permissions so that last answer wasn't the correct one. `read_only_keys` does kind of make sense as a name, but I don't love it because you aren't actually reading any values here. These aren't fields which you are reading but not writing, they are essentially keys you are skipping. Maybe something like `ignore_update_keys`?

Reply:
I see where you're going. `ignore_update_keys` sounds like 2 actions in 1. 
maybe `non_editable_keys` .. `immutable_keys` .. `dont_touch_these_keys`